app/frontend/styles/components/flashes.scss

```diff
@@ -40,6 +40,10 @@
 
 .govuk-notification--notice {
   border-color: govuk-colour('blue');
+
+  .govuk-notification__list a {
+    color: govuk-colour('blue');
```

Comment:
Isn't blue the default for <a>? I'd be tempted to just specify when you need it to not be the default color

Reply:
Indeed it is, copy pasta got out of hand 🍝